usecases/build/build.go

```diff
@@ -0,0 +1,42 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2024 Weaviate B.V. All rights reserved.
+//
+//  CONTACT: hello@weaviate.io
+//
+
+package build
+
+import (
+	"runtime"
+
+	"github.com/prometheus/common/version"
+)
+
+// These package level variables are set during build time via ldflags -X
+// Also these information are passed to prometheus version package so that it's available in nice `weaviate_info` metric
+var (
+	Version   string
+	Revision  string
+	Branch    string
+	BuildUser string
+	BuildDate string
+	GoVersion string
```

Comment:
I think it would be great to add comments on each setting what it means

Reply:
Done.